Dockerfile.server

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------------------
```

Comment:
I would like to reuse the building stage, which has been copied&pasted from the `Dockerfile`. Ideas?